### PR TITLE
Revise what-is-kubevela

### PR DESCRIPTION
**Pull request by resouer**
As title said. kubevela is a CD system.

### CODE REVIEW

**Review by zzxwill — APPROVED** [review]
/lgtm

File: docs/getting-started/introduction.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ The typical examples are Heroku and Cloud Foundry. They provide full application
 
 Though the biggest difference lies in **flexibility**.
 
-KubeVela does not introduce any restriction. As a plus, even its deployment workflow and full feature set are implemented as LEGO-sytle CUE modules and can be extended at any time when your needs grow. Comparing to this mechanism, traditional PaaS systems are highly restricted, i.e. they have to enforce constraints in the type of supported applications and capabilities, and as application needs grows, you always outgrow the capabilities of the PaaS system - this will never happen in KubeVela platform.
+KubeVela is fully programmable. Its deployment workflow steps and even full feature set are all LEGO-sytle CUE modules and can be added/removed/re-programmed in-place when your needs change. Comparing to this mechanism, traditional PaaS systems are highly restricted, i.e. they have to enforce constraints in the type of supported applications and capabilities, and as application needs grows, you always outgrow the capabilities of the PaaS system - this will never happen in KubeVela platform.
```
Comment (hongchaodeng) [inline docs/getting-started/introduction.md]:
```suggestion
KubeVela is fully programmable. All of its deployment workflow and component feature set are LEGO-style CUE modules and can be extended or removed in-place when your needs change. Comparatively, traditional PaaS systems are highly restricted, i.e. they have to enforce constraints in the supported types of applications and capabilities -- That means as the platform grows, you always outgrow the capabilities of the PaaS system. Growing and extending is so much easier to do in the KubeVela platform.
```

File: versioned_docs/version-v1.1/getting-started/introduction.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ The typical examples are Heroku and Cloud Foundry. They provide full application
 
 Though the biggest difference lies in **flexibility**.
 
-KubeVela does not introduce any restriction. As a plus, even its deployment workflow and full feature set are implemented as LEGO-sytle CUE modules and can be extended at any time when your needs grow. Comparing to this mechanism, traditional PaaS systems are highly restricted, i.e. they have to enforce constraints in the type of supported applications and capabilities, and as application needs grows, you always outgrow the capabilities of the PaaS system - this will never happen in KubeVela platform.
+KubeVela is fully programmable. Its deployment workflow steps and even full feature set are all LEGO-sytle CUE modules and can be added/removed/re-programmed in-place when your needs change. Comparing to this mechanism, traditional PaaS systems are highly restricted, i.e. they have to enforce constraints in the type of supported applications and capabilities, and as application needs grows, you always outgrow the capabilities of the PaaS system - this will never happen in KubeVela platform.
```
Comment (hongchaodeng) [inline versioned_docs/version-v1.1/getting-started/introduction.md]:
```suggestion
KubeVela is fully programmable. All of its deployment workflow and component feature set are LEGO-style CUE modules and can be extended or removed in-place when your needs change. Comparatively, traditional PaaS systems are highly restricted, i.e. they have to enforce constraints in the supported types of applications and capabilities -- That means as the platform grows, you always outgrow the capabilities of the PaaS system. Growing and extending is so much easier to do in the KubeVela platform.
```

**Review by hongchaodeng — APPROVED** [review]
lgtm with comment

**Review by wonderflow — APPROVED** [review]

**Pull request merged**